README.md

```diff
@@ -17,6 +17,21 @@ docker build --tag provideq-toolbox-backend --build-arg GAMS_LICENSE=$(base64 -w
 docker run --publish 8080:5000 provideq-toolbox-backend
 ```
 
+## Releasing a new version
+1. Create a release branch from develop: `git checkout -b release/x.y.z`.
+2. Bump the version number in the `build.gradle` file to the new version number and commit it to the release branch.
+3. Push to GitHub and create a pull request to merge the release branch into `main`.
+4. Make sure to test your new version!
+5. Write a changelog.
+   The PR can help you identify differences between the last release (`main`) and the next one (your release branch).
+6. Tag the release commit with your version number and attach the changelog as its message: `git tag -a x.y.z`
+7. Merge the PR into main.
+8. Push your new tag (`git push --tags`) and create a new GitHub release from it:
```

Comment:
Isn't it easier to let GitHub handle all of the tag creation stuff? Afaik a tag automatically gets created when you create a new GitHub release.

Reply:
I didn't know that GitHub can do this. I've updated the docs!